src/drivers/http.ts

```diff
@@ -24,37 +32,23 @@ export default defineDriver((opts: HTTPOptions) => {
         headers: { ...opts.headers, ...topts.headers },
       })
         .then(() => true)
-        .catch(() => false);
+        .catch((err) => catchFetchError(err, false));
     },
     async getItem(key, tops = {}) {
-      try {
-        return await _fetch(r(key), {
-          headers: { ...opts.headers, ...tops.headers },
-        });
-      } catch (err: any) {
-        if (err?.response?.status === 404) {
-          return null;
-        }
-
-        throw err;
-      }
+      const value = await _fetch(r(key), {
+        headers: { ...opts.headers, ...tops.headers },
+      }).catch(catchFetchError);
+      return value;
```

Comment:
Is there a reason you prefer to assign a variable instead of simply returning?

Reply:
`return await` could also do it similarly (it is for making stack traces better in some runtimes)

`value` + `return` is mostly conventional across drivers to make debugging easier